src/components/pages/search/search.tsx

```diff
@@ -12,12 +13,21 @@ import './search.scss';
 const Search: FC = () => {
   const { globalSearch } = useContext(StoreContext);
 
+  useEffect(() => {
+    globalSearch?.triggerFilterRequest();
+  }, [])
+
   return (
     <div
       className="search"
       data-component="page/search"
     >
       <div className="search__results-area">
+        <SearchResultNavigation
+          hits={globalSearch?.hits}
```

Comment:
An sich spricht nichts dagegen `hits`, `size` und `from` der Komponente als Props/Parameter mit rein zu reichen.
In der Komponente wird aber für die Actions zudem noch auf den StoreContext zugegriffen.
Hier müsste man abwägen, ob man entweder alles über Props reinreicht, oder über den StoreContext innerhalb der Komponente auf den Store zugreift.